tests/controltab/test_config_view.py

```diff
@@ -0,0 +1,97 @@
+# This file is part of ts_m2gui.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import pytest
+import logging
+
+from lsst.ts.m2gui import Model
+from lsst.ts.m2gui.controltab import TabConfigView
+
+
+@pytest.fixture
+def widget(qtbot):
+
+    widget = TabConfigView("Configuration View", Model(logging.getLogger()))
+    qtbot.addWidget(widget)
+
+    return widget
+
+
+def test_callback_signal_config(qtbot, widget):
+
+    widget.model.report_config(
+        file_configuration="a",
+        file_version="b",
+        file_control_parameters="c",
+        file_lut_parameters="d",
+        power_warning_motor=1.1,
+        power_fault_motor=1.2,
+        power_threshold_motor=1.3,
+        power_warning_communication=1.4,
+        power_fault_communication=1.5,
+        power_threshold_communication=1.6,
+        in_position_axial=1.7,
+        in_position_tangent=1.8,
+        in_position_sample=1.9,
+        timeout_sal=2.1,
+        timeout_crio=2.2,
+        timeout_ilc=3.0,
+        misc_range_angle=3.1,
+        misc_diff_enabled=True,
+        misc_range_temperature=3.2,
+    )
+
+    assert widget._label_file_configuration.text() == "File of Configuration: a"
```

Comment:
This looks much better in a QFormLayout, where configuration name is QLabel for the row, and value is another QLabel.

Reply:
Fixed in the commit: [29ca788](https://github.com/lsst-ts/ts_m2gui/pull/6/commits/29ca788f3c2b709d92963b35534482e61bb71a11). Thanks!